<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DoctorFormClass</class>
 <widget class="QMainWindow" name="DoctorFormClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>838</width>
    <height>445</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>DoctorForm</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout_3">
    <item row="0" column="0" rowspan="2">
     <widget class="QListWidget" name="listWidgetPatients"/>
    </item>
    <item row="0" column="1">
     <widget class="QGroupBox" name="groupBox">
      <property name="title">
       <string>Add patient</string>
      </property>
      <layout class="QGridLayout" name="gridLayout">
       <item row="0" column="0">
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>Name:</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1" colspan="3">
        <widget class="QLineEdit" name="lineEditName"/>
       </item>
       <item row="1" column="0" colspan="2">
        <widget class="QLabel" name="label_3">
         <property name="text">
          <string>Diagnosis:</string>
         </property>
        </widget>
       </item>
       <item row="1" column="2" colspan="2">
        <widget class="QLineEdit" name="lineEditDiagnosis"/>
       </item>
       <item row="2" column="0" colspan="3">
        <widget class="QLabel" name="label_4">
         <property name="text">
          <string>Specialisation:</string>
         </property>
        </widget>
       </item>
       <item row="2" column="3">
        <widget class="QLineEdit" name="lineEditSpecialisation"/>
       </item>
       <item row="3" column="0">
        <widget class="QLabel" name="label_5">
         <property name="text">
          <string>Doctor:</string>
         </property>
        </widget>
       </item>
       <item row="3" column="2" colspan="2">
        <widget class="QLineEdit" name="lineEditDoctor"/>
       </item>
       <item row="4" column="0" colspan="3">
        <widget class="QLabel" name="label">
         <property name="text">
          <string>Admission date:</string>
         </property>
        </widget>
       </item>
       <item row="4" column="3">
        <widget class="QDateEdit" name="dateEdit"/>
       </item>
       <item row="5" column="0" colspan="4">
        <widget class="QPushButton" name="pushButtonAdd">
         <property name="text">
          <string>Add </string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item row="1" column="1" rowspan="2">
     <widget class="QGroupBox" name="groupBox_2">
      <property name="title">
       <string>Update patient</string>
      </property>
      <layout class="QGridLayout" name="gridLayout_2">
       <item row="0" column="0">
        <widget class="QLabel" name="label_7">
         <property name="text">
          <string>Diagnosis:</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1" colspan="2">
        <widget class="QLineEdit" name="lineEditUpdateDiagnosis"/>
       </item>
       <item row="1" column="0" colspan="2">
        <widget class="QLabel" name="label_6">
         <property name="text">
          <string>Specialisation:</string>
         </property>
        </widget>
       </item>
       <item row="1" column="2">
        <widget class="QLineEdit" name="lineEditUpdateSpecialisation"/>
       </item>
       <item row="2" column="0" colspan="3">
        <widget class="QPushButton" name="pushButtonUpdate">
         <property name="text">
          <string>Update</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QCheckBox" name="checkBox">
      <property name="text">
       <string>Only your patients</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="DoctorForm.qrc"/>
 </resources>
 <connections/>
</ui>
